src/librustdoc/html/static/themes/dark.css

```diff
@@ -386,3 +386,8 @@ kbd {
 		background: #353535;
 	}
 }
+
+.hidden-by-impl-hider, 
```

Comment:
This rule has nothing to do in themes. Please put in `rustdoc.css` at the upper level.